ariadne_codegen/plugins/base.py

```diff
@@ -0,0 +1,11 @@
+import ast
+
+from graphql import GraphQLSchema
+
+
+class BasePlugin:
```

Comment:
We could call this just "Plugin"

Reply:
Done. I have thought about raising `NotImplementedError` but decided against it, it would be annoying to implement a plugin that only uses a small number of hooks/methods.